google-chrome-launcher

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+chrome_flatpak_dir=`flatpak-spawn --host flatpak info -l com.google.Chrome`
+chrome_binary=${chrome_flatpak_dir}/files/extra/opt/google/chrome/google-chrome
+
+exec flatpak-spawn --host --env=LD_LIBRARY_PATH=${chrome_flatpak_dir}/files/lib ${chrome_binary}
```

Comment:
We should do some $* or we'll break the methods in the desktop file, or other scripts that rely on passing arguments in to the browser.

Reply:
Yes, I had this in a later version of the patch. Pushed here now.